app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -1,3 +1,4 @@
+import { useContext } from 'react';
 import { PageHeading } from '@18f/identity-components';
```

Comment:
> I did not see the new Review examples of how to take clear photos of your ID message when I used image_resolution_error.yml

@night-jellyfish 
(I am continuing our discussion about the above point here, because it's easier to reply to github conversations and mark them as resolved when the conversation is associated with a specific file.)
I manually tested with `image_resolution_error.yml` and I did see the `Review examples of how to take clear photos of your ID` link.  
![10538HelpCenterLInkAfterRefactor](https://github.com/18F/identity-idp/assets/80347702/195edfce-a351-47f7-ac03-840b9a80d2b4)

Reply:
I tried again and this time I saw the text! I'm not sure what happened earlier. Thanks for following up.